catalogue_tools/b_value/tests/test_estimate_beta.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+
+# import functions from other modules
+from catalogue_tools.utils.simulate_distributions import simulate_magnitudes
+
+# import functions to be tested
+from catalogue_tools.b_value.estimate_beta import estimate_beta_tinti
+from catalogue_tools.b_value.estimate_beta import estimate_beta_utsu
+from catalogue_tools.b_value.estimate_beta import estimate_beta_elst
+
+
+def test_estimate_beta_tinti():
+    n = 1000000
+    beta = 1 / np.log10(np.e)
```

Comment:
For simplicity I'd maybe say beta=np.log(10)?

Reply:
agree!